presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -811,6 +811,14 @@ Optimizer Properties
 
     Track history based plan statistics from complete plan fragments in failed queries.
 
+``optimizer.log-plans-used-in-hbo``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Suggest changing the heading formatting to the same character length as the character length in line 814.